docs/schemaform/form-config.md

```diff
@@ -51,6 +52,11 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
       // JSON schema object for the page. Follows the JSON Schema format.
       schema: {
         type: 'object',
+        // In a schema's properties, there are references to definitions. For example:
+        //   "homePhone": { "$ref": "#/definitions/phone" }
+        // In the config file, the definition for phone would then need to be added into definitions
+        // for it to be parsed correctly and added to homePhone.
+        definitions: {},
```

Comment:
@raquelromano-gov How would you suggest modifying it?

From what I understand, the schema we're importing from `vets-json-schema` is what's used to share schema between the front and back ends, but unless we put it in the config in `config/form.js`, it's not used. The trouble with piping some of the `properties` in the schema from `vets-json-schema` directly into the config is they have `$ref`s to `"#/definitions/something"`, so when `react-json-schema` (I think?) finds that ref, it looks in this `definitions` object (, not in the schema from `vets-json-schema`, and if we don't have the definition in _here_ (a page's individual schema in `config/form.js`), it'll error out.

The way @jbalboni explained it to me is every page is a separate schema as far as `react-json-schema` is concerned. What that means is we need to fill in everything that it needs to make it a complete json schema per the standard.

I really don't know how to explain it very well, and I feel like I may be missing your point entirely! I'm open to suggestions.